tests/meta/workflow/test_analysis.py

```diff
@@ -89,7 +89,6 @@ def test_is_flow_cell_check_applicable(mip_analysis_api: MipDNAAnalysisAPI, anal
     # GIVEN that no samples are down-sampled nor external
     for sample in case.samples:
         assert not sample.from_sample
-        assert not sample.is_external
```

Comment:
sample.is_external is a property though? So why should this be deleted?

Reply:
Because it is not used

Reply:
it is used in the delivery? or is that based on the app tag?

Reply:
Oh, sorry, I thought it was a property which returned `sample.application_version.application.is_external`. My bad. We would probably want that as a property though 😅 But that is for another PR

Reply:
Yes